interp/dumpglob.mli

```diff
@@ -36,8 +36,8 @@ val dump_binding : ?loc:Loc.t -> Names.Id.Set.elt -> unit
 val dump_notation :
   (Constrexpr.notation * Notation.notation_location) Loc.located ->
   Notation_term.scope_name option -> bool -> unit
-val dump_constraint :
-  Vernacexpr.typeclass_constraint -> bool -> string -> unit
+
+val dump_constraint : Names.Name.t Loc.located -> bool -> string -> unit
```

Comment:
No way to preserve the abstraction given by the type (e.g. by defining it at worst in the holdall file `misctypes.ml`)? Not a big deal otherwise.